shared/translations/default.json

```diff
@@ -0,0 +1,19 @@
+{
+  "Home": "Home",
+  "Recently updated": "Recently updated",
+  "Recently viewed": "Recently viewed",
+  "Created by me": "Created by me",
+  "Profile saved": "Profile saved",
+  "Profile picture updated": "Profile picture updated",
+  "Unable to upload new avatar": "Unable to upload new avatar",
```

Comment:
```suggestion
  "Unable to upload new profile picture": "Unable to upload new profile picture",
```

Reply:
Again - CAn't be changed

Reply:
I realize it shouldn't be changed here, but in the source file?

Reply:
Yes sure, i will do it